whistleblower/contracts-kotlin/src/main/kotlin/net/corda/examples/whistleblower/contracts/BlowWhistleContract.kt

```diff
@@ -7,7 +7,7 @@ import net.corda.core.contracts.requireThat
 import net.corda.core.transactions.LedgerTransaction
 import net.corda.examples.whistleblower.states.BlowWhistleState
 
-const val BLOW_WHISTLE_CONTRACT_ID = "net.corda.examples.whistleblower.contracts.BlowWhistleContract"
+const val BLOW_WHISTLE_CONTRACT_ID = "BlowWhistleContract"
 
```

Comment:
I dont think you can shorten this. This fails when I try to run it. 
With this error: Cannot find contract attachments for BlowWhistleContractnull. See https://docs.corda.net/api-contract-constraints.html#debugging